packages/app-account/src/containers/Account.js

```diff
@@ -54,20 +52,19 @@ const mapStateToProps = (state, ownProps) => {
     isAttentionMoreInfoModalVisible
   }
 
+  if (!selectors.isChainLoaded(state, { chainAlias: PRIVATE })) {
+    return notAuthenticating
+  }
+
   const {
```

Comment:
Handling the redirect parameters will be pulled out into a separate component in a separate PR.